templates/content/oe-organisation--default.html.twig

```diff
@@ -0,0 +1,21 @@
+<h4 class="fw-bold mb-3">
+  {% if elements.oe_acronym is not empty %} {{ elements.oe_acronym }}{% endif %}
+</h4>
+{% set _items = [] %}
+{% for _field in elements if _key|first != '#' %}
+  {% set _value %}
+    {{ _field|field_value }}
+  {% endset %}
+  {% if _value is not empty %}
```

Comment:
The emptiness check is not working.
If you create stakeholders where some of the fields are empty, you get empty html elements.
Use `{% if _value|trim is not empty %}`, that works.